votes/initiateNewVote/_EDIT_ME.yml

```
@@ -24,8 +24,9 @@ headerInstructions: |
 #    voters express their preference for each candidates, no matter how many
 #    there are.
 candidates:
-  - TODO
-  - TODO
+  - Status Quo: keep distributing `corepack` with Node.js, disabled by default, exactly as it is today.
+  - Keep distributing `corepack` with Node.js, enabling `yarn` and `pnpm` by default.
+  - Stop distributing `corepack` with Node.js.
```

Let’s be clear that enabling Yarn and pnpm means creating executables for them.

```suggestion
  - Status Quo: keep distributing Corepack with Node.js, disabled by default, exactly as it is today.
  - Keep distributing Corepack with Node.js and include in the Node.js distribution `yarn` and `pnpm` executables that run Corepack to download and run Yarn and pnpm.
  - Stop distributing Corepack with Node.js.
```

---

This should be applied since it's important to make people aware